grafana-mimir.yaml

```diff
@@ -18,23 +18,17 @@ pipeline:
     with:
       repository: https://github.com/grafana/mimir
       tag: mimir-${{package.version}}
-      expected-commit: 2db2c4d9de8565c52fcef3050b95b8a2808b407e
+      expected-commit: 24e4281c138d873772076c5ac276a0f20f633d0d
 
   # the repo has an ssh://git@github.com URI for mimir-prometheus
   # so just rm the file and default to the https URL for go.mod
   - runs: rm .gitconfig
 
-  - uses: go/bump
-    with:
-      deps: golang.org/x/crypto@v0.31.0 golang.org/x/net@v0.33.0
-      modroot: .
-
   - uses: go/build
     with:
       modroot: .
       packages: ./cmd/mimir
       output: grafana-mimir
-      ldflags: "-extldflags -static -s -w"
 
   - uses: strip
```

Comment:
@Dentrax we can also consider removing strip because go/build already does the stripping. 💭